### PR TITLE
ADEN-6579 Remove Badge Ad code

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/provider/remnantGpt.js [hunk 1]
```diff
@@ -28,10 +28,6 @@ define('ext.wikia.adEngine.provider.remnantGpt', [
 		{
 			afterSuccess: function (slotName) {
 				slotTweaker.removeDefaultHeight(slotName);
-				if (!uapContext.isBfaaLoaded()) {
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/provider/remnantGpt.js:31]: `uapContext` is unused now

**Comment by kvas-damian** [comment]
Can we also remove https://github.com/Wikia/app/blob/d97176608d9b3e071e206fee9a9e5d4035552652/extensions/wikia/AdEngine/js/AdEngine2.js#L177-L177 ?

File: extensions/wikia/AdEngine/js/SlotTweaker.js [hunk 2]
```diff
@@ -71,28 +69,6 @@ define('ext.wikia.adEngine.slotTweaker', [
 		}
 	}
 
-	// TODO: remove it after fully disabling Badge Ad: ADEN-6579
-	function adjustLeaderboardSize(slotname) {
-		var slot = doc.getElementById(slotname);
-		if (!geo.isProperGeo(instantGlobals.wgAdDriverDisableBadgeAdCountries) &&
-			isTopLeaderboard(slotname) && isStandardLeaderboardSize(slotname)) {
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/SlotTweaker.js:78]: `isStandardLeaderboardSize()` looks like it's not used now

**Comment by kvas-damian** [comment]
Please grep for `TOP_BUTTON_WIDE`. I found some leftovers.

File: extensions/wikia/AdEngine/js/provider/remnantGpt.js (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/provider/remnantGpt.js:31]: Right, removed.

File: extensions/wikia/AdEngine/js/SlotTweaker.js (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/SlotTweaker.js:78]: Yep, removed too.

**Comment by dmnsobczak** [comment]
`slotTweaker.hide('TOP_BUTTON_WIDE')` - removed
`TOP_BUTTON_WIDE` - styles, logic and template includes removed
`isTopLeaderboard()` - now unused, removed

File: extensions/wikia/AdEngine/js/provider/directGpt.js [hunk 3]
```diff
@@ -52,10 +52,6 @@ define('ext.wikia.adEngine.provider.directGpt', [
 		{
 			afterSuccess: function (slotName) {
 				slotTweaker.removeDefaultHeight(slotName);
-				if (!uapContext.isBfaaLoaded()) {
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/provider/directGpt.js:55]: `uapContext` is unused now

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/provider/directGpt.js:55]: I keep forgetting that. Removed from code and tests.

**Review by kvas-damian — APPROVED** [review]

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 4]
```diff
@@ -48,7 +48,6 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverDelayCountries';
 		$vars[] = 'wgAdDriverDelayTimeout';
 		$vars[] = 'wgAdDriverDisableSraCountries';
-		$vars[] = 'wgAdDriverDisableBadgeAdCountries';
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: Removing variable from IG at this moment can enable Badge Ad on cached pages. Please bring it back here ;)

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: That's true. Restored, creating new blocked task to remember to remove that after deploy.

**Pull request merged**